TestApp/Scripts/experience.js

```diff
@@ -26,6 +27,99 @@ function CreateSpheresAsync() {
     return Promise.resolve();
 }
 
+function CreatePlane(width, height, uvScale) {
```

Comment:
Use BABYLON.Mesh.CreatePlane?

Reply:
At the worst case, you can use VertexData.CreatePlane and strip out the indices before using it.

Reply:
I see. Perhaps you can load one of the glTF asset generator models to test this? https://github.com/KhronosGroup/glTF-Asset-Generator/tree/master/Output/Positive/Mesh_PrimitiveMode

Reply:
and I also need scaled uv... I'll move that function into its own test js

Reply:
Sounds good.